jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/identity/DBUserGroupCallbackImplTest.java

```diff
@@ -34,12 +41,34 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
 
+@RunWith(Parameterized.class)
 public class DBUserGroupCallbackImplTest {
 
+    public enum Configuration {
```

Comment:
maybe we can restrict a little access modifier here, not a big issue. Maybe `private` or `protected`?

Reply:
Switched to protected